roles/language/tasks/main.yml

```diff
@@ -1,9 +1,13 @@
 ---
-- name: install language package(s)
-  sudo: yes
-  apt: pkg={{ item }} state=latest force=yes
-  with_items: language_packages
+#- name: install language package(s)
+#  sudo: yes
+#  apt: pkg={{ item }} state=latest force=yes
+#  with_items: language_packages
```

Comment:
Probably didn't mean to push this through. I [documented](http://www.mzoo.org/deploying-openfoodnetwork-to-staging-on-aws-ec2/) having done it when deploying to AWS, but not very clearly. So as with most of these changes, probably best to ignore and not spend any more time on it than I've already wasted in my haste in ignorance. : )